_api_app/app/Http/routes.php

```diff
@@ -16,24 +16,24 @@
 // });
 
 // @@@:TODO: Require login for API endpoints
-$app->group(['prefix' => 'v1','namespace' => 'App\Http\Controllers'], function($app) {
-	$app->get('state/{site}','StateController@get');
+$app->group(['prefix' => 'v1', 'namespace' => 'App\Http\Controllers'], function ($app) {
+    $app->get('state/{site}', 'StateController@get');
 
-    $app->patch('update-site','SiteController@update');
-    $app->post('create-site','SiteController@create');
-    $app->delete('delete-site/{site}','SiteController@delete');
-    $app->put('order-sites','SiteController@order');
+    $app->post('site', ['as' => 'site', 'uses' => 'SiteController@create']);
```

Comment:
we should use plural when naming endpoints, because all of them can work with multiple items.

Reply:
Fixed in: https://github.com/berta-cms/berta/pull/108/commits/0eae3d586608b6d6d97ff7cb6077f99e69247a37